src/Gaffer/Reference.cpp

```diff
@@ -199,6 +248,7 @@ class Reference::PlugEdits : public boost::signals::trackable
 		struct PlugEdit
```

Comment:
What defines something as a "PlugEdit" conceptually?  Anything done to the plug of a reference other than a setValue()?

Reply:
Conceptually, anything done to the plug after it has been loaded, so anything not authored in the reference file itself. In practice, we only track a subset of those things currently. I've added a comment to clarify this in bba39a1.